tt_metal/hw/inc/grayskull/tensix_types.h

```diff
@@ -213,6 +213,7 @@ enum class DataFormat : std::uint8_t {
     Bfp2_b = 15,
     Lf8 = 10,
     Int8 = 14,
+    UInt8 = 30,
```

Comment:
how do you know it is 30?

Reply:
also, @aliuTT why remove the PR template? fill it in please

Reply:
Filled in 🫡

Reply:
30 masks to 14 (Int8) for hardware conversions. This is in line with WH. We should add asserts for GS hardware conversions for Int8 and Uint8 for GS, since neither is supported in hardware. Separate issue opened to track this: https://github.com/tenstorrent/tt-metal/issues/9646